packages/aws-cdk-lib/aws-globalaccelerator-endpoints/lib/nlb.ts

```diff
@@ -14,6 +14,19 @@ export interface NetworkLoadBalancerEndpointProps {
    * @default 128
    */
   readonly weight?: number;
+
+  /**
+   * Forward the client IP address in an `X-Forwarded-For` header
+   *
+   * GlobalAccelerator will create Network Interfaces in your VPC in order
+   * to preserve the client IP address.
+   *
+   * Client IP address preservation is supported only in specific AWS Regions.
+   * See the GlobalAccelerator Developer Guide for a list.
+   *
+   * @default true if available
```

Comment:
According to the documentation, my understanding is that the default value is `false` : 
https://docs.aws.amazon.com/global-accelerator/latest/dg/preserve-client-ip-address.html
> When you add an Network Load Balancer with security groups as an endpoint in Global Accelerator, client IP address preservation is not enabled by default.

```suggestion
   * @default false
```